Assets/MixedRealityToolkit.Services/InputSimulation/Implementation/Microsoft.MixedReality.Toolkit.Services.InputSimulation.Editor.asmdef

```diff
@@ -1,15 +1,20 @@
-{
-    "name": "Microsoft.MixedReality.Toolkit.Services.InputAnimation.Editor",
-    "references": [
-        "Microsoft.MixedReality.Toolkit",
-        "Microsoft.MixedReality.Toolkit.Editor.Utilities",
-        "Microsoft.MixedReality.Toolkit.Services.InputAnimation",
-        "Microsoft.MixedReality.Toolkit.Services.InputSimulation"
-    ],
-    "optionalUnityReferences": [],
-    "includePlatforms": [
-        "Editor"
-    ],
-    "excludePlatforms": [],
-    "allowUnsafeCode": false
+{
+    "name": "Microsoft.MixedReality.Toolkit.Services.InputSimulation.Implementation",

```

Comment:
>InputSimulation [](start = 53, length = 15)

I think this file is still called "InputSimulation.Editor" but it doesn't match the name of the asmdef - I think this file should be renamed to InputSimulation.Implementation

Reply:
I've moved files back into Editor for the time being, can do a clean file move later for git history.